### DISCUSSION TITLE
Broken go install command on README

### DISCUSSION THREAD

**Discussion by fortuna** [prometheus/prometheus#12078] [Q&A]
### What did you do?

I ran this command from the [README](https://github.com/prometheus/prometheus#building-from-source):
```
GO111MODULE=on go install github.com/prometheus/prometheus/cmd/...
```

### What did you expect to see?

The `prometheus` command line tool to be successfully installed, so I could call `$(go env GOPATH)/bin/prometheus`

### What did you see instead? Under which circumstances?

I got the error: 
```
pattern github.com/prometheus/prometheus/cmd/...: github.com/prometheus/prometheus@v0.37.0: missing go.sum entry
```

### System information

Linux 5.18.16-1rodete4-amd64 x86_64

**Comment by alex-petrov-vt** [comment]
Hmm, works for me on the latest main:

```
❯ prometheus --version
prometheus, version  (branch: , revision: e1b708200853371517480176da0a03437b3bb2c2)
  build user:
  build date:
  go version:       go1.18
  platform:         linux/amd64

❯ uname -a
Linux alex-laptop 5.4.0-131-generic #147-Ubuntu SMP Fri Oct 14 17:07:22 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by LeviHarrison** [comment]
@fortuna did you execute the command in the root of the cloned repository? I've found that you need to do that to get it to work, although it could be a different issue.
